serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-openapi-based-services.adoc

```diff
@@ -0,0 +1,399 @@
+Orchestration of OpenAPI based services
+=======================================
+:compat-mode!:
+// Metadata:
+:description: Orchestration of OpenAPI based services
+:keywords: kogito, workflow, serverless, openapi, services, json
+// links:
+:swagger_inspector_url: https://swagger.io/docs/swagger-inspector/how-to-create-an-openapi-definition-using-swagger/
+:quarkus_swagger_url: https://quarkus.io/guides/openapi-swaggerui
+:spring_swagger_url: https://springdoc.org/#Introduction
+:go_swagger_url: https://goswagger.io/
+:python_flasgger_url: https://github.com/flasgger/flasgger
+:dotnet_swagger_url: https://docs.microsoft.com/en-us/aspnet/core/tutorials/web-api-help-pages-using-swagger?view=aspnetcore-6.0
+:npm_swagger_url: https://www.npmjs.com/package/swagger-ui-express
+:php_swagger_url: https://github.com/zircote/swagger-php
+// Referenced documentation pages.
+:getting-familiar-with-our-tooling: xref:getting-started/getting-familiar-with-our-tooling.adoc
+:create-your-first-workflow-service: xref:getting-started/create-your-first-workflow-service.adoc
+:build-workflow-image-with-quarkus-cli: xref:cloud/build-workflow-image-with-quarkus-cli.adoc
+:understanding-jq-expressions: xref:core/understanding-jq-expressions.adoc
+:configuring-openapi-services-endpoints: xref:service-orchestration/configuring-openapi-services-endpoints.adoc
+:camel-k-integration: xref:integrations/camel-k-integration.adoc
+:authention-support-for-openapi-services: xref:security/authention-support-for-openapi-services.adoc
+
+This document describes how to call REST services via an link:{open_api_spec_url}[OpenAPI] specification file.
+
+The examples in this document assume that you have the Quarkus tooling installed and have created a Serverless Workflow project. 
+
+For more information about the tooling, see {getting-familiar-with-our-tooling}[Using Serverless Workflow tooling]. To create a new project, see {create-your-first-workflow-service}[Creating your first Serverless Workflow service].
+
+== The OpenAPI Function Definition
+
+Kogito follows the {spec_doc_url}#using-functions-for-restful-service-invocations[CNCF Serverless Workflow specification function definition] to make calls to remote REST services using an OpenAPI specification reference, as shown in the following example:
+
+.An example of an OpenAPI function definition
+[source,json]
+----
+{
+   "functions":[
+      {
+         "name":"myFunction1",
+         "operation":"file:///home/users/kogito/myopenapi-file.yaml#myFunction1"
+      }
+   ]
+}
+----
+
+The `operation` attribute is a URI composed by:
```

Comment:
Hmmm, I do not think this is, strictly speaking, 100% accurated
Actuall the operation is a string (not an URI) which must contain a # separator (yes I know the # is a valid URI fragment, but in other operation types, there are multiple #, and you cannot have multiple fragments in an URI, so I think is better to not use URI for the whole string, because it wont be consistent with other operations strings) 
The # separator separated an URI (which is used to locate the OpenAPI definition)  from the operation Id (which is the part of the OpenAPI definition that identifies the REST call to be performed)
And serverles workflow URis supports the three schemes